README.md

```diff
@@ -15,6 +15,8 @@ address a common need for cloud native development:
    - with the ability to select a specific subset of events from a particular
      producer.
 
+For the full mission of Knative Eventing see [docs/mission.md](./docs/mission.md).
```

Comment:
It seems like the 10 or 25-word mission statement should probably exist on the front page here and knative.dev.

Reply:
done.